jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -151,36 +151,60 @@ def run_tf(mode):
       else:
         assert False
 
-    def is_tf_exception(lim: correctness_stats.Limitation):
-      return (lim.error_type == 'Missing TF support' and
+    def expected_missing_tf_support(lim: correctness_stats.Limitation):
+      return (lim.error_type == correctness_stats.CATEGORY_MISSING_TF_SUPPORT and
+              self.tf_default_device.device_type in lim.devices)
+    def expected_possible_incorrect(lim: correctness_stats.Limitation):
```

Comment:
nit: maybe we can be more consistent between the name of the function here and the name of the category?